west.yml

```diff
@@ -210,11 +210,6 @@ manifest:
       path: modules/debug/TraceRecorder
       groups:
         - debug
-    - name: trusted-firmware-m
-      path: modules/tee/tfm
-      revision: c74be3890c9d975976fde1b1a3b2f5742bec34c0
-      groups:
-        - tee
```

Comment:
We've talked about this a bit, would it make more sense to comment these lines out and add an additional comment line pointing to the GitHub Issue rather than just remove them?

I'm thinking more along the lines of giving someone who needs TF-M enough breadcrumbs to know why it was removed (not just release notes) and enough runway to know the challenges they're about to encounter with trying to enable it.

Reply:
+1

Reply:
I added the link to this pr, where we have full context.